.ci/Jenkinsfile

```diff
@@ -43,108 +43,6 @@ pipeline {
         }
       }
     }
-    stage('Install') {
```

Comment:
Removing Install, build, and lint stages too, as `lint` is moved to test.yml and Release stage does `npm ci` and `npm run build` within its stage.